speed-dreams/source-2.2.3/src/simulatedDrivingAssistance/backend/SocketBlackBox.inl

```diff
@@ -78,6 +78,9 @@ void SocketBlackBox<BlackBoxData, PointerManager>::Initialize(BlackBoxData& p_in
     msgpack::pack(sbuffer, data);
     m_server.SendData(sbuffer.data(), sbuffer.size());
 
+    m_server.AwaitData(m_buffer, SBB_BUFFER_SIZE);
+    if (m_buffer[0] != 'O' && m_buffer[1] != 'K') throw std::exception("Black box send wrong message: OK expected");
```

Comment:
AND should be OR if anything other than OK is wrong
KK -> true && false -> false -> no error